README.md

```diff
@@ -161,12 +171,6 @@ Publish an event
 What features will be added soon?
 ====================================
 
-- Release a Core
-    DELETE /v1/devices/:coreid
-
-- Claim a core
-    POST /v1/devices
-
 - per-user / per-core ownership and access restrictions.  Right now ANY user on your local cloud can access ANY device.
```

Comment:
not sure about this.. it works for devices, but not for other entities, like webhooks/events..so don't delete this for now.